tests/manage/monitoring/prometheusmetrics/test_rgw.py

```diff
@@ -25,6 +30,7 @@
 @tier4c
 @pytest.mark.polarion_id("OCS-2385")
 @skipif_managed_service
+@runs_on_provider
```

Comment:
Is this marker correct?

Reply:
Yes

Reply:
earlier marker Reference https://github.com/red-hat-storage/ocs-ci/blob/e73ff0c0638ae1e1e0acc2aee18888c5e59717aa/tests/manage/pv_services/test_del_mon_service_and_create_pvc.py#L55